README.md

```diff
@@ -6,189 +6,95 @@
-        secrets:
-          - source: quote_txt
-            target: quote.txt
-    ```
-
-## Updating your container ##
-
-### Docker Compose ###
-
-1. Pull the new image from Docker Hub:
-
-    ```console
-    docker compose pull
-    ```
-
-1. Recreate the running container by following the [previous instructions](#running-with-docker-compose):
-
-    ```console
-    docker compose up --detach
-    ```
-
-### Docker ###
-
-1. Stop the running container:
-
-    ```console
-    docker stop <container_id>
-    ```
-
-1. Pull the new image:
+## Monitoring ##
 
-    ```console
-    docker pull cisagov/example:0.0.1
-    ```
+The following web services are started for monitoring the underlying components:
 
-1. Recreate and run the container by following the [previous instructions](#running-with-docker).
+- Celery Flower: [http://localhost:5555](http://localhost:5555)
+- Mongo Express: [http://localhost:8083](http://localhost:8083)
+- Redis Commander: [http://localhost:8082](http://localhost:8082)
 
 ## Image tags ##
 
 The images of this container are tagged with [semantic
-versions](https://semver.org) of the underlying example project that they
-containerize.  It is recommended that most users use a version tag (e.g.
-`:0.0.1`).
+versions](https://semver.org) of the [admiral](https://github.com/cisagov/admiral)
+Python library that they containerize.  It is recommended that most users
+use a version tag (e.g. `:0.0.1`).
 
 | Image:tag | Description |
 |-----------|-------------|
-|`cisagov/example:1.2.3`| An exact release version. |
-|`cisagov/example:1.2`| The most recent release matching the major and minor version numbers. |
-|`cisagov/example:1`| The most recent release matching the major version number. |
-|`cisagov/example:edge` | The most recent image built from a merge into the `develop` branch of this repository. |
-|`cisagov/example:nightly` | A nightly build of the `develop` branch of this repository. |
-|`cisagov/example:latest`| The most recent release image pushed to a container registry.  Pulling an image using the `:latest` tag [should be avoided.](https://vsupalov.com/docker-latest-tag/) |
-
-See the [tags tab](https://hub.docker.com/r/cisagov/example/tags) on Docker
+|`cisagov/admiral-docker:1.2.3`| An exact release version. |
+|`cisagov/admiral-docker:1.2`| The most recent release matching the major and minor version numbers. |
+|`cisagov/admiral-docker:1`| The most recent release matching the major version number. |
+|`cisagov/admiral-docker:edge` | The most recent image built from a merge into the `develop` branch of this repository. |
+|`cisagov/admiral-docker:nightly` | A nightly build of the `develop` branch of this repository. |
+|`cisagov/admiral-docker:latest`| The most recent release image pushed to a container registry.  Pulling an image using the `:latest` tag [should be avoided.](https://vsupalov.com/docker-latest-tag/) |
+
+See the [tags tab](https://hub.docker.com/r/cisagov/admiral-docker/tags) on Docker
```

Comment:
Use correct image name.  Also, versions in the README should match the value in src/version.txt:
```suggestion
|`cisagov/admiral:1.3.0`| An exact release version. |
|`cisagov/admiral:1.3`| The most recent release matching the major and minor version numbers. |
|`cisagov/admiral:1`| The most recent release matching the major version number. |
|`cisagov/admiral:edge` | The most recent image built from a merge into the `develop` branch of this repository. |
|`cisagov/admiral:nightly` | A nightly build of the `develop` branch of this repository. |
|`cisagov/admiral:latest`| The most recent release image pushed to a container registry.  Pulling an image using the `:latest` tag [should be avoided.](https://vsupalov.com/docker-latest-tag/) |

See the [tags tab](https://hub.docker.com/r/cisagov/admiral/tags) on Docker
```